VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionPostsAdapter.java

```diff
@@ -80,22 +81,26 @@ public void render(BaseViewHolder tag, DiscussionThread discussionThread) {
         {
             final String text;
             final Iconify.IconValue icon;
+            final boolean iconAtStart;
```

Comment:
What's the point of creating this extra variable which is assigned the same value as `voteCountsEnabled`?